var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -47,10 +47,10 @@ def install(self, spec, prefix):
         if '+lmsensors' in spec:
             with working_dir("src/components/lmsensors"):
                 configure_args = [
-                    "--with-sensors_incdir=%s/include/sensors" %
-                    spec['lm-sensors'].prefix,
-                    "--with-sensors_libdir=%s/lib" %
-                    spec['lm-sensors'].prefix]
+                    "--with-sensors_incdir=%s/sensors" %
+                    spec['lm-sensors'].headers.directories[0],
```

Comment:
Can you check, I think this might actually return `prefix.include.sensors` so the `sensors` above may not be necessary.

Reply:
It only returns prefix.include.  I was going to modify lm-sensors "headers()" to return prefix.include.sensors using find_headers(), but it would always return just the include directory.